epub33/core/index.html

```diff
@@ -2873,128 +2932,87 @@ <h6>Examples</h6>
 </pre>
 								</aside>
 
-								<aside class="example" id="example-manifest-flbk">
-									<p>The following example shows a <code>manifest</code> that references two
-											<a>Foreign Resources</a> and therefore uses the <a
-											href="#sec-foreign-restrictions-manifest">fallback chain mechanism</a> to
-										supply content alternatives. The fallback chain terminates with a <a>Core Media
-											Type Resource</a>.</p>
-									<pre>&lt;manifest&gt;
-    &lt;item id="item1" 
-          href="chap1_docbook.xml" 
-          media-type="application/docbook+xml" 
-          fallback="fall1"/&gt;
-    &lt;item id="fall1" 
-          href="chap1.xml" 
-          media-type="application/z3998-auth+xml" 
-          fallback="fall2" /&gt;
-    &lt;item id="fall2" 
-          href="chap1.xhtml" 
-          media-type="application/xhtml+xml"/&gt; 
-    … 
+								<aside class="example" id="example-manifest-flbk"
+									title="Foreign Resource with Fallback in Spine">
+									<p>The following example shows the <a href="#sec-foreign-restrictions-manifest"
+											>fallback chain mechanism</a> allowing a <a>Foreign Resource</a> (JPEG) to
+										be listed in the spine with fallback to an SVG Content Document.</p>
+									<pre>Spine:
+&lt;itemref idref="page-001"/&gt;
+
+Manifest:
+&lt;manifest&gt;
+   &lt;item id="page-001"
+      href="images/page-001.jpg"
+      media-type="image/jpeg"
+      fallback="#page-001-svg"/&gt;
+
+   &lt;item id="page-001-svg"
+      href="images/page-001.svg"
+      media-type="image/svg+xml"/&gt;
+      … 
 &lt;/manifest&gt;
 </pre>
 								</aside>
 
-								<aside class="example">
-									<p>The following example shows a reference to a remote audio file that EPUB Creators
-										must reference from the manifest (Reading Systems will render the audio inline
-										in the XHTML Content Document, so the file is a Publication Resource).</p>
+								<aside class="example" title="Foreign Resource with Fallback">
+									<p>The following example shows a hyperlink to a JPEG that will render as a
+											<a>Top-Level Content Document</a>. As a result, EPUB Creators must list the
+										JPEG as a non-linear item in the <a href="#sec-spine-elem">spine</a>
+										<em>and</em> specify a fallback to an EPUB Content Document in its manifest
+										declaration (in this case, an SVG).</p>
 									<pre>XHTML:
-&lt;audio src="http://www.example.com/book/audio/ch01.mp4" controls="controls"/&gt;
+&lt;img src="images/infographic.jpg" alt="&#8230;" />
```

Comment:
What is the role of this entry as far as the example is concerned? My understanding is that by virtue of referring to the image via a `<a>` the item must be added to the spine as non-linear, etc. I.e., the presence of `<img>` seems to be irrelevant for the example...

Reply:
It was asked in the issue for a realistic example of it in use. The image shows the context - that it's not embedding that requires a spine entry. I can expand on that in the description.

Reply:
Well, let us see what the expansion leads to :-) because I do not see why having the `<img>` makes it more realistic... As far as I am concerned, the idea that a content links "out" to an image, ie, the author wants the image to be the "main" content on the screen is perfectly realistic...

Reply:
The examples were put in to instruct people on when resources have to be listed in the manifest/spine and when they don't. Showing a hyperlink to an image without showing how the image is rendered within the current content document is cutting off half the issue. You're always expanding an image that's been displayed, but it's not the image itself in the content that is the issue, it's the use of a hyperlink. It's better to be explicit about that than leave people guessing about whether it only applies to the hyperlink.

Reply:
Ok, I take that, but then let us make this explicit in the example caption... I think main issue to emphasize is

>  it's not the image itself in the content that is the issue, it's the use of a hyperlink

Reply:
Example 27 bothers me. Why would we require a fallback? The image is not referenced from the spine. 

In most reading systems, if you click or double-click on an image it will open in its own window anyway. We don't need a manifest fallback in those cases. Why do we need one here?

Reply:
> The image is not referenced from the spine.

The image has to be referenced from the spine if you hyperlink to it. It's a top-level content document in that case.

> In most reading systems, if you click or double-click on an image it will open in its own window anyway

Ya, but this is an effort to come up with something authors might plausibly do that would load in the spine. It used to be a link to open an audio file, but that's more unrealistic. In this case, presumably the idea is that this ensures the image opens as a separate document in all reading systems regardless of extra functionality.

I found at least one old epubcheck complaint about this: https://github.com/w3c/epubcheck/issues/324

And I'm sure if I had the inclination to search the old IDPF forums there would be threads there, too, as this is a nuisance for authors that has popped up from time-to-time over the years.

But if you have a better example of non-standard stuff being linked to in the spine, I'm open to anything else.

Reply:
> let us make this explicit in the example caption

I did a rewrite earlier this morning, but I've tweaked it some more, including the example title. If there's something you think still needs saying, let me know.

Reply:
@mattgarrish the example looks o.k. to me, and I am happy to approve the PR.

However... I wonder whether we should not separate this PR from the genuine issue @dauwhe has raised about the usefulness of this case (see also https://github.com/w3c/epub-specs/issues/1464#issuecomment-970504419). What about changing the example for a link to an external data file like, say, a CSV file or a Javascript raw code? That is of course not a realistic example for a novel, but it is for a scholarly book. In both cases providing a fallback does make more sense (although I am not sure any RS implements it) and it separates from the issue whether a fallback on a core media type should be required in the first place... And it would allow us to merge this otherwise fine PR.

Reply:
> I wonder whether we should not separate this PR from the genuine issue @dauwhe has raised about the usefulness of this case

Ya, but are we likely to change this revision? I'm not a fan of fallbacks, but allowing CMTs in the spine is a big change in direction to squeeze into the last few weeks before going to CR, especially after already going through the accessibility horizontal review.

If we change to CSV, then the example would no longer be able to explain why embedded content is different from hyperlinked content, since you can't embed a CSV without a fallback, either. The only things you can embed are CMTs, so we'd just be shuffling media types.

Reply:
On the CSV issue: for me, the essence of the example is that if a direct link is done to a CSV file from the EPUB, and that file is part of the EPUB instance, then a fallback should be provided (at least this is the way I understand things). Maybe that should be a separate example, which does not mix in the hyperlink vs. embedding and helps people understand the issue.

B.t.w., I do not think @dauwhe is considering allowing an image in the spine for a linear content without fallback.

Reply:
> if a direct link is done to a CSV file from the EPUB, and that file is part of the EPUB instance, then a fallback should be provided

Arguably, no, but we never solved "downloadable" content from an EPUB.

Linking to a CSV within the EPUB is kind of unrealistic, as getting a mess of raw data dumped into the reading system display area probably doesn't help users, especially if the RSes ignore it and render an HTML fallback, which I presume would format the data as a table. But as a second example I don't really care that strongly about realism, since we're out on the edges anyway.

> I do not think @dauwhe is considering allowing an image in the spine for a linear content without fallback.

I'm not sure that changes the issue much, as we don't definitively define behaviour for linear v. non-linear.

Reply:
Are you okay with the new example, @iherman ? (i.e., can we merge this?)

Reply:
Yes, I am, that is why I gave a thumbs up :-)

Reply:
Okay, wasn't sure if that was for the idea of adding the example or for what I committed.